scripts/tosdr.js

```diff
@@ -50,12 +50,27 @@ function getSitePoints (sites) {
         for (pointName in pointsData) {
             let point = pointsData[pointName]
             let pointCase = point.tosdr.case
+            let score = point.tosdr.score || 0
             if (!pointCase) continue
-            
+
+            // standardize case (some of them start with caps)
+            pointCase = pointCase.toLowerCase()
+            // standardize score (some of them come as strings)
+            score = parseInt(score, 10)
+
             let type = point.tosdr.point
 
             if (type === 'good' || type === 'bad')
-                addPoint(points, type, pointCase, point.tosdr.score)
+                addPoint(points, type, pointCase, score)
+        }
+
+        // we use class in our score but we may not have privacy-related reasons for it
+        // so show all available reasons instead
+        if (points.class &&
+                (!points.match.good || !points.match.good.length) &&
+                (!points.match.bad || !points.match.bad.length)) {
+            points.match.good = points.all.good
+            points.match.bad = points.all.bad
```

Comment:
There's an edge case where we get a class from TOSDR, but no privacy-related points. This falls back on displaying all TOSDR points so the user can see why we gave it that class. See e.g. seenthis.net:

![screenshot from 2018-01-22 18-35-16](https://user-images.githubusercontent.com/1431577/35237585-036fea8e-ffa3-11e7-8d17-b1b7a2267a76.png)